app/assets/javascripts/select-kit/components/user-tracking-dropdown.js.es6

```diff
@@ -0,0 +1,76 @@
+import DropdownSelectBox from "select-kit/components/dropdown-select-box";
+
+export default DropdownSelectBox.extend({
+  pluginApiIdentifiers: [""],
```

Comment:
I think this line is unnecessary.

Reply:
Yea, I left here just to get thoughts, what do we need or use this line for?